Description: Mark the Maven dependencies as optional to match the package dependencies and the POM deployed on Maven Central
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: no
--- a/pom.xml
+++ b/pom.xml
@@ -207,27 +207,32 @@
        <groupId>ant</groupId>
        <artifactId>ant</artifactId>
         <version>1.6.5</version>
+       <scope>provided</scope>
    </dependency>
    <dependency>
        <groupId>ant</groupId>
        <artifactId>ant-launcher</artifactId>
         <version>1.6.5</version>
 		<scope>test</scope>
+       <scope>provided</scope>
    </dependency>
     <dependency>
         <groupId>bcel</groupId>
         <artifactId>bcel</artifactId>
          <version>5.1</version>
+        <optional>true</optional>
     </dependency>
     <dependency>
         <groupId>xerces</groupId>
         <artifactId>xercesImpl</artifactId>
          <version>2.8.1</version>
+        <optional>true</optional>
     </dependency>	
     <dependency>
         <groupId>commons-httpclient</groupId>
         <artifactId>commons-httpclient</artifactId>
          <version>3.0.1</version>
+        <optional>true</optional>
     </dependency>	
 	<!--  could not find a copy of Ivy in M2's repository,
 	      download http://www.jayasoft.org/downloads/ivy/1.4.1/ivy-1.4.1.jar
@@ -237,6 +242,7 @@
         <groupId>jayasoft</groupId>
         <artifactId>ivy</artifactId>
          <version>1.4.1</version>
+       <scope>provided</scope>
     </dependency>	
   <dependency>
     <groupId>junit</groupId>
